test/params/str-list.spec.js

```diff
@@ -6,6 +6,22 @@ describe('Testing StrList Parameter', () => {
   const jsonParam = api.StrList('list', 'json');
   const queryParamEnum = api.StrList('list', 'query', { enums: ['enumOne', 'enumTwo', 'enumThree'] });
   const bodyParamEnum = api.StrList('list', 'json', { enums: ['enumOne', 'enumTwo', 'enumThree'] });
+  const queryParamInValidConstraints = api.StrList('list', 'query', {
+    minListItemLength: 1,
+    maxListItemLength: 5,
+    maxListLength: 3
+  });
+  const bodyParamInValidConstraints = api.StrList('list', 'json', {
+    minListItemLength: 1,
+    maxListItemLength: 5,
+    maxListLength: 3
+  });
+  const queryParamInValidMin = api.StrList('list', 'query', { minListItemLength: 1 });
+  const bodyParamInValidMin = api.StrList('list', 'json', { minListItemLength: 1 });
+  const queryParamInValidMax = api.StrList('list', 'query', { maxListItemLength: 5 });
+  const bodyParamInValidMax = api.StrList('list', 'json', { maxListItemLength: 5 });
+  const queryParamInValidLength = api.StrList('list', 'query', { maxListLength: 2 });
+  const bodyParamInValidLength = api.StrList('list', 'json', { maxListLength: 2 });
```

Comment:
Wrap each of these in their own describe, it's messy to have to jump up and down from the tests to the definitions to see why / if a test should be passing or failing